src/main/res/layout/activity_advanced_card_info.xml

```diff
@@ -92,7 +92,7 @@
         android:padding="8dip"
         android:background="#FF8800"
         android:visibility="gone"
-        android:text="@string/unknown_card_description"
```

Comment:
This is used.

Reply:
I'm pretty sure that it gets replaced when actually loading this activity but let's remove those hunks, at least for now.